### PR TITLE
Brian/#54/min1 day available

### LINKED ISSUE #54: Reject events that have no days flagged as available

### PR DESCRIPTION
**Pull request by brian-e-haley**
Closes #64 .

### CODE REVIEW

**Comment by brian-e-haley** [comment]
Need to reexamine Events PUT and handle Client side bug.

**Comment by gerardopar** [comment]
this is for handling days when they are all false?

**Comment by brian-e-haley** [comment]
@gerardopar that is correct this is to ensure on any `POST` or `PUT` requests if all of the days are set to `False` a `400` error will be returned.

File: server/project/api/events_handler.py [hunk 1]
```diff
@@ -9,6 +9,17 @@
...
+
+def verify_at_least_1_day_available(availability: Availability) -> int:
+    """Counts the number of days selected as available and returns the number
+    as an integer."""
+    days_available = 0
+    for day in availability['days'].values():
+        if day:
+            days_available += 1
+    return days_available
+
```
Comment (Kcheung42) [inline server/project/api/events_handler.py]: Could also do 

`return any(availability['days`].values()`

File: server/project/api/events_handler.py [hunk 2]
```diff
@@ -9,6 +9,17 @@
 from project.error_handlers import *
 import datetime as dt
 
+
+def verify_at_least_1_day_available(availability: Availability) -> int:
+    """Counts the number of days selected as available and returns the number
```
Comment (Kcheung42) [inline server/project/api/events_handler.py]: Will this function break if availability is `None` ?

Comment (brian-e-haley) [reply server/project/api/events_handler.py]: `if 'availability' in data and not verify_at_least_1_day_available(data['availability']):
    raise NoDayAvailable`

File: server/project/api/events_handler.py (hunk 1 above)
Comment (brian-e-haley) [reply server/project/api/events_handler.py]: Implemented thank you!

**Pull request merged**